src/main/resources/application.yml

```diff
@@ -112,6 +112,10 @@ interventions-delius:
       "[Finance, Benefits and Debt (FBD)]": CRS03
       "[Dependency and Recovery (D&R)]": CRS04
 
+delius-nsi:
+  requirement:
+    rehabilitation-activity-type: F
+
```

Comment:
Very minor (so leave it as is), but IMHO since this is not environment specific not sure it gains from being in the config when it could just be a hardcoded constant, just more levels of indirection when trying to understand what is going on.